dts/bindings/i2s/nxp,mcux-i2s.yaml

```diff
@@ -5,7 +5,7 @@ description: NXP mcux SAI-I2S controller
 
 compatible: "nxp,mcux-i2s"
 
-include: [i2s-controller.yaml]
+include: [i2s-controller.yaml, pinctrl-device.yaml]
```

Comment:
suggestion here: in general you'll need at least `pinctrl-0` and `pinctrl-names`, so making them required is a good idea (will generate a better error message), ST does that for example.